doc/api/process.md

```diff
@@ -1963,32 +1969,50 @@ A boolean value that is `true` if the current Node.js build includes support for
 
 <!-- YAML
 added: v4.8.0
+deprecated: REPLACEME
 -->
 
+> Stability: 0 - Deprecated. Use `process.features.tls` instead.
+
 * {boolean}
 
 A boolean value that is `true` if the current Node.js build includes support for ALPN in TLS.
 
+Since OpenSSL now unconditionally supports ALPN, this value is identical to that of
+`process.features.tls`.
+
 ## `process.features.tls_ocsp`
 
 <!-- YAML
 added: v0.11.13
+deprecated: REPLACEME
 -->
 
+> Stability: 0 - Deprecated. Use `process.features.tls` instead.
+
 * {boolean}
 
 A boolean value that is `true` if the current Node.js build includes support for OCSP in TLS.
 
+Since OpenSSL now unconditionally supports OCSP, this value is identical to that of
+`process.features.tls`.
+
 ## `process.features.tls_sni`
 
 <!-- YAML
 added: v0.5.3
+deprecated: REPLACEME
 -->
 
+> Stability: 0 - Deprecated. Use `process.features.tls` instead.
+
 * {boolean}
 
 A boolean value that is `true` if the current Node.js build includes support for SNI in TLS.
 
+Since OpenSSL now unconditionally supports SNI, this value is identical to that of
```

Comment:
the word `now` is ephemeral and may not be good at API doc, can you pls use something like `since vx.y or similar?